### DISCUSSION TITLE
23.11.30 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#80] [Daily Scrum]
.

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - PR 피드백 받은 내용 수정
  - 친구 추가 알림 기능 연동
  - 플레이리스트 음악 저장 기능 구현
- 내일은 어떤 일을 하나요?
  - 플레이리스트 음악 저장 기능 구현 마무리
  - 플레이리스트 알림 기능 연동

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 친구 기능 테스트를 위해 문서화 시도
    - 테스트 커버리지를 높이기 위해 jacoco를 활용할 수 있다.. 
- 내일은 어떤 일을 하나요?
    - 문서화하며 발견한 예외 상황 처리
    - 테스트 작성
